MastodonSDK/Sources/MastodonUI/ViewModel/RelationshipViewModel.swift

```diff
@@ -263,6 +268,10 @@ extension RelationshipViewModel {
             optionSet.insert(.showReblogs)
         }
 
+        if isDomainBlocking {
+            optionSet.insert(.domainBlocking)
```

Comment:
I'm wondering if we, somewhere need to remove this attribute incase a domain gets unblocked? However the App likely won'T notice this anyway properly so maybe this is going away with the `MastodonUser` refactoring anyways.